generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx.ejs

```diff
@@ -49,11 +50,17 @@ import { getLoginUrl } from 'app/shared/util/url-utils';
 <%_ if (enableTranslation) { _%>
 import { locales } from 'app/config/translation';
 <%_ } _%>
+import { getProfile } from 'app/shared/reducers/profile';
 import appConfig from 'app/config/constants';
 
 export interface IHeaderProps {
   isAuthenticated: boolean;
   isAdmin: boolean;
+  activeProfiles: string[];
+  ribbonEnv: string;
+  isInProduction: boolean;
```

Comment:
you just need `isInProduction` and `isSwaggerEnabled` right, plz remove props that are not used

Reply:
Shouldn't I keep `ribbonEnv` too ? It's use to set the ribbon content, like in Angular:
```
 <Translate contentKey={`global.ribbon.${this.props.ribbonEnv}`}>{this.props.ribbonEnv}</Translate>
```

Reply:
yes, i didnt notice it